framework/src/media/audio/audio_manager.c

```diff
@@ -144,13 +172,15 @@ audio_manager_result_t find_input_audio_card(void)
 
 audio_manager_result_t find_output_audio_card(void)
 {
+	int ret;
 	unsigned int card_id = -1;
 	unsigned int device_id = -1;
 	char type;
 	struct dirent *dir_entry;
 	DIR *dir_info;
+	char path[256] = "/dev/audio/";
```

Comment:
same comment regarding configs.

Reply:
Okay. Let me add the config in another PR..